src/Classes/GemSelectControl.lua

```diff
@@ -581,11 +581,8 @@ function GemSelectClass:AddCommonGemInfo(gemInstance, grantedEffect, addReq, mer
 		if cost then
 			self.tooltip:AddLine(16, "^x7F7F7FCost: ^7"..cost)
 		end
-		if grantedEffectLevel.soulCost then
-			self.tooltip:AddLine(16, string.format("^x7F7F7FSouls Per Use: ^7%d", grantedEffectLevel.soulCost))
-		end
-		if grantedEffectLevel.skillUseStorage then
-			self.tooltip:AddLine(16, string.format("^x7F7F7FCan Store ^7%d ^x7F7F7FUse (%d Souls)", grantedEffectLevel.skillUseStorage, grantedEffectLevel.skillUseStorage * grantedEffectLevel.soulCost))
+		if grantedEffectLevel.vaalStoredUses then
```

Comment:
Would it be possible to format it like this in the order. Would need special handling again for the soul costs to look the same and moving cooldown time before the soul stuff.
![image](https://user-images.githubusercontent.com/31533893/222422625-458c56c3-add9-4bca-89be-ae8a873f20f5.png)